compass/ocean/tests/drying_slope/drying_slope.cfg

```diff
@@ -0,0 +1,27 @@
+# Options related to the vertical grid
+[vertical_grid]
+
+# Number of vertical levels
+vert_levels = 10
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = sigma
+
+# config options for drying slope test cases
+[drying_slope]
+
+# the number of grid cells in x and y
+nx = 6
+ny = 28
+
+# the size of grid cells (m)
+dc = 1000.0
+
+# config options for visualizing drying slope ouptut
+[drying_slope_viz]
+
+# frames per second for movies
+frames_per_second = 30
+
+# movie format
+movie_format = mp4
```

Comment:
What do you think about an option to skip making the movie?

Reply:
Hmm, that's not a bad idea.  The default should probably be to *not* make the movie if we're using this as a regression test.

Reply:
Good idea. I'll add that.